src/Components/OrdersDisplay/SingleOrderDisplay.js

```diff
@@ -178,7 +180,7 @@ export default function SingleOrderDisplay({ orderDetails, span }) {
               </p>
             </div>
           </div>
-          <div className={`px-3 py-1 border-2 border-t-0 rounded-b-lg ${orderDetails.channel === "En salle" ? "bg-yellow-100" : orderDetails.channel === "A emporter" ? "bg-blue-100" : "bg-purple-100"}`} style={{ columnCount: span }}>
+          <div className={`${selectOrder === index ? 'border-4 border-orange-500 shadow-lg' : ''} px-3 py-1 border-2 border-t-0 rounded-b-lg ${orderDetails.channel === "En salle" ? "bg-yellow-100" : orderDetails.channel === "A emporter" ? "bg-blue-100" : "bg-purple-100"}`} style={{ columnCount: span }}>
```

Comment:
- [ ] No shadow in the mock-up
- [ ] Not right color
- [ ] The content of the order shouldn't move when selected